packages/cli/src/transformers.ts

```diff
@@ -315,7 +322,13 @@ export function getGlobalsTransformer({ typeChecker }: TransformerOptions) {
 
       if (insertShim) {
         return factory.updateSourceFile(modifiedSourceFile, [
-          getNamespaceImport(shimsIdentifier, ESM_SHIMS_PACKAGE),
+          getDirnameHelperFunction(dirnameHelperFunctionName),
```

Comment:
Do we always need all the shims?

Reply:
Not the surname function, but everything else yeah. I can add a case for that